qemu/tests/hotplug_virtio_mem.py

```diff
@@ -0,0 +1,52 @@
+import time
+
+from virttest import error_context
+
+from virttest.utils_misc import normalize_data_size
+from virttest.utils_test.qemu import MemoryHotplugTest
+from provider import virtio_mem_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Virtio-mem hotplug test
+    1) Boot guest
+    2) Hotplug virtio-mem device
+    3) Check virtio-mem device
+    4) Resize virtio-mem device twice
+    5) Check virtio-mem device
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    timeout = params.get_numeric("login_timeout", 240)
+    threshold = params.get_numeric("threshold", target_type=float)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login(timeout=timeout)
+    node_id = params.get_numeric("node_memory_vmem0", 1)
+
+    target_mem = params.get("target_mems")
+    hotplug_test = MemoryHotplugTest(test, params, env)
+    hotplug_test.hotplug_memory(vm, target_mem)
+
+    device_id = "virtio_mem-%s" % target_mem
```

Comment:
We can use a more better way to decouple with avocado-vt about device id format.
Suggest:
```
vmem_dev =  hotplug_test.hotplug_memory(vm, target_mem)
device_id = vmem_dev.get_qid()
 device_id = vmem_dev.get_qid()
```

Reply:
@yanan-fu 

Well, the [hotplug_memory function return a list of devices](https://github.com/avocado-framework/avocado-vt/blob/cb61c3975cf8bf6d025b3f08a3caebbef094bb68/virttest/utils_test/qemu/__init__.py#L654), including the existing memory objects, so I guess I will have to obtain the virtio-mem device following an approach like the previously implemented:

`vm.devices.get_by_params({'driver': virtio_mem_model})`

Reply:
You are right, it return both memory object and the virtio mem device,  it can be implement with:
```
_,  vmem_dev = hotplug_test.hotplug_memory(vm, target_mem)
```

Reply:
@yanan-fu thanks a lot for the suggestion, changed now!